tests/load/filesystem/test_filesystem_client.py

```diff
@@ -50,6 +50,9 @@ def test_filesystem_destination_configuration() -> None:
 @pytest.mark.parametrize("layout", ALL_LAYOUTS)
 def test_successful_load(write_disposition: str, layout: str, default_buckets_env: str) -> None:
     """Test load is successful with an empty destination dataset"""
+    if default_buckets_env.startswith("gdrive://"):
+        pytest.skip("GDrive is not supported for this test")
```

Comment:
Temporary skip until the GDrive starts to work as a destination as well

Reply:
I think it would be better to create another fixture along `default_buckets_env` that contains gdrive and use it in dict test. ie `source_buckets_env` and then remove gdrive from default. otherwise you'll need to add this condition to all destination tests

Reply:
Hm, sounds cunning. Done